src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -67,7 +75,39 @@ protected override bool ValidateConfiguration(DiscordNotificationSettings settin
                 Message = parsed.Message,
             };
 
+            string userString = TvRequest.RequestedUser.Alias;
             notification.Other.Add("image", parsed.Image);
+            if (Customization.ApplicationUrl.HasValue()) notification.Other.Add("authorUrl", Customization.ApplicationUrl + "requests");
+
+            if (model.RequestType == RequestType.Movie)
+            {
+                if (mentionAlias)
+                {
+                    notification.Other.Add("mention", userString);
+                    userString = TvRequest.RequestedUser.UserName;
+                }
+                notification.Other.Add("footer", "Requested by " + MovieRequest.RequestedUser + " on " + MovieRequest.RequestedDate.ToLongDateString());
+                notification.Other.Add("author", "🎬 New Movie Request!");
+                notification.Other.Add("title", MovieRequest.Title + " (" + MovieRequest.ReleaseDate.Year + ")");
+                notification.Other.Add("description", MovieRequest.Overview);
+                notification.Other.Add("titleUrl", IMDB_BASE_URL + MovieRequest.ImdbId);
+                Logger.LogDebug(MovieRequest.Background);
```

Comment:
Why are we logging the background? Might show strange in the logs with no context